src/types/docs/README.md

```diff
@@ -0,0 +1,325 @@
+# README
+
+## Top-level Schemas
+
+*   [Defender](./defender.md) – `-`
+
+*   [Definitions](./definitions.md) – `-`
+
+*   [Function](./function.md) – `-`
+
+*   [Provider](./provider.md) – `-`
+
+*   [Resources](./resources.md) – `-`
+
+## Other Schemas
+
+### Objects
+
+*   [Autotask](./definitions-definitions-autotask.md) – `undefined#/definitions/autotask`
+
+*   [BlockSentinel](./definitions-definitions-blocksentinel.md) – `undefined#/definitions/blockSentinel`
+
+*   [Contract](./definitions-definitions-contract.md) – `undefined#/definitions/contract`
+
+*   [DatadogConfig](./definitions-definitions-datadogconfig.md) – `undefined#/definitions/datadogConfig`
+
+*   [EmailConfig](./definitions-definitions-emailconfig.md) – `undefined#/definitions/emailConfig`
+
+*   [FortaSentinel](./definitions-definitions-fortasentinel.md) – `undefined#/definitions/fortaSentinel`
+
+*   [Notification](./definitions-definitions-notification.md) – `undefined#/definitions/notification`
+
+*   [Policy](./definitions-definitions-policy.md) – `undefined#/definitions/policy`
+
+*   [Relayer](./definitions-definitions-relayer.md) – `undefined#/definitions/relayer`
+
+*   [Sentinel](./definitions-definitions-sentinel.md) – `undefined#/definitions/sentinel`
+
+*   [TelegramConfig](./definitions-definitions-telegramconfig.md) – `undefined#/definitions/telegramBotConfig`
+
+*   [Trigger](./function-properties-trigger.md) – `undefined#/properties/trigger`
+
+*   [Untitled object in Definitions](./definitions-definitions-relayer-properties-address-from-relayer.md) – `undefined#/definitions/relayer/properties/address-from-relayer`
```

Comment:
Do we know if there's a setting for removing the "untitled" items from the generated documentation? If not, I'd add a post-processing step on the mds with a regex that just removes these entries.

Reply:
Doesn't seem to be a setting for it, I've added a step to remove this after the build process.